internal/format/testdata/markdown/table-WithRequired.golden

```diff
@@ -40,26 +40,26 @@ followed by another line of text.
 
 | Name | Version |
```

Comment:
Instead of modifying `WithRequired` golden file, you can create a new golden file (and its corresponding item in test map for `WithAnchor`. And Please do that for:

- markdown document
- markdown table
- asciidoc document
- asciidoc table